CMakeLists.txt

```diff
@@ -47,7 +47,6 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 if(AMR_WIND_ENABLE_CUDA)
   enable_language(CUDA)
   # Fix issues with GPU builds
```

Comment:
Let's remove this comment too then.

Reply:
I haven't tested this with a GPU build+ninja.  Should we try that first?

Reply:
No. If we don't need these, then they should be removed regardless.